### PR TITLE
Bug 1207646 - Fix failure in test_delete_call_log.py

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=760d984c9579509e698f232e8b57ad3b8afad50e

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=12ccd9ae48a0d19b00b39e63ff06b322dd9e6e90

File: tests/python/gaia-ui-tests/gaiatest/mocks/mock_call.py [hunk 1]
```diff
@@ -4,20 +4,19 @@
...
-    def __init__(self, phone_number=None, call_type='incoming', date=date.today(), duration=1,
+    def __init__(self, phone_number=None, call_type='incoming', date=datetime.date.today(), duration=1,
                  status='cancelled', serviceId=0, emergency=False, voicemail=False, **kwargs):
         super(MockCall, self).__init__(self, **kwargs)
 
         current_time = repr(time.time()).replace('.', '')
         self['number'] = phone_number if phone_number is not None else '555%s' % current_time[-7:]
         self['type'] = call_type
-        self['date'] = '{:%m/%d/%Y}'.format(date)
+        self['date'] = int(time.mktime(datetime.datetime.strptime('{:%m/%d/%Y}'.format(date), '%m/%d/%Y').timetuple()) * 1000)
```
Comment (npark-mozilla) [inline tests/python/gaia-ui-tests/gaiatest/mocks/mock_call.py]: if you want it a bit simpler, following would work as well:
self['date'] = int(time.mktime(date.timetuple())*1000)

Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/mocks/mock_call.py]: Thanks, seems to work

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=50fd879af54237db0a2fcdea7c7ef8f08580dad2

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b3169c317ba26ada9f8e1871713f887ff4e83184

**Pull request merged**